src/theme/src/theme-tools.test.js

```diff
@@ -1,74 +1,149 @@
-import { getValue, resolveThemeTokens } from './theme-tools'
+import { getValue, mergeTheme, resolveThemeTokens } from './theme-tools'
 
 describe('Theme tools', () => {
-  it('getValue returns the value based on a path lookup', () => {
-    const obj = {
-      foo: {
-        bar: 'baz'
+  describe('getValue', () => {
+    it('returns the value based on a path lookup', () => {
+      const obj = {
+        foo: {
+          bar: 'baz'
+        }
       }
-    }
 
-    const result = getValue(obj, 'foo.bar')
-    expect(result).toEqual('baz')
-  })
+      const result = getValue(obj, 'foo.bar')
+      expect(result).toEqual('baz')
+    })
 
-  it('getValue returns the value as-is if the path cannot be resolved', () => {
-    const obj = {
-      foo: {
-        bar: 'baz'
+    it('returns the value as-is if the path cannot be resolved', () => {
+      const obj = {
+        foo: {
+          bar: 'baz'
+        }
       }
-    }
 
-    const result = getValue(obj, '12px')
-    expect(result).toEqual('12px')
+      const result = getValue(obj, '12px')
+      expect(result).toEqual('12px')
+    })
   })
 
-  it('resolveThemeTokens preserves nested object structures', () => {
-    const theme = {}
-    const props = {
-      baseStyle: {
-        color: 'blue'
-      },
-      appearances: {
-        primary: {
-          backgroundColor: 'pink'
+  describe('mergeTheme', () => {
```

Comment:
I typically wouldn't write tests for a function that's just wrapping a 3rd party util, but they were quick to spin up and may prevent a regression if someone were to swap it with a plain object spread/assign